### DISCUSSION TITLE
What's going on with release changes in latest releases?

### DISCUSSION THREAD

**Discussion by guhcampos** [argoproj/argo-cd#16250] [General]
The last few releases have not published the list of changes correctly. Both `2.8.5` and `2.8.6` don't mention what was fixed, one would need to go over the git diffs to figure those out, which in all honesty is not feasible for most of us to which ArgoCD is one of the many tools we need to be aware on our organizations.

Those were minor releases though, but `2.9.0` has also been published with no release notes, which is a bit weirder because the release candidates did have changes published on GitHub. At this point, if you followed the RCs you could have an idea of what's included in `2.9.0` but most people can't be assumed to have followed the RCs.

Even more, there's no documentation for upgrading to `2.9.0` at all, which means the release was tagged without the proper documentation being created. I did some browsing over the RCs and I can't find any drafts of such docs either, which means those are definitely missing and not unpublished.

**Comment by KevinBonilla** [comment]
Agreed, as an end consumer the only way I could understand what changes are being made is if I look through each RC which is confusing and not user friendly. Would like to see minor releases with full changelogs going forward.
